lib/screens/health/health_screen.dart

```diff
@@ -137,68 +136,18 @@ class _HealthScreenState extends State<HealthScreen> {
                           Row(
                             children: [
                               HealthCard(
-                                  title: 'Crash Free Users',
-                                  value: viewModel.projects[_index].release?.crashFreeUsers,
-                                  change: 0.04
-                              ), // TODO(denis): Use delta from api, https://github.com/getsentry/sentry-mobile/issues/11
+                                  title: 'Stability Score',
```

Comment:
We are renaming this to `Crash Free` on web for now - I think we should do the same here.